core/commands/checkout.py

```diff
@@ -126,10 +126,19 @@ class GsCheckoutCurrentFileAtCommitCommand(LogMixin, WindowCommand, GitCommand):
     Reset the current active file to a given commit.
     """
 
+    show_commit_info = False
+
     def run(self):
         if self.file_path:
             super().run(file_path=self.file_path)
 
+    def on_highlight(self, commit):
+        if commit:
+            self.window.run_command('gs_show_file_diff', {
+                'commit_hash': commit,
+                'fpath': self.file_path
```

Comment:
use file_name

Reply:
Beside, that I don't know which thing I should rename here. You use `file_path` throughout your code to refer to full paths, whereby `file_name` refers to a naked filename without its path. What I mean is, I did not invent this name here and I get `self.file_path` from the inherited code.

Reply:
Ops. I meant `file_path` instead of fpath.